src/components/executive-education-2u/utils.js

```diff
@@ -1,5 +1,7 @@
+import moment from 'moment';
 import { PAID_EXECUTIVE_EDUCATION } from '../course/data/constants';
 import { getActiveCourseRun } from '../course/data/utils';
+import { DATE_FORMAT } from '../program/ProgramCourses';
```

Comment:
Should we use `DATE_FORMAT` from `src/components/course/data/constants.js` rather than a constant from an unrelated "programs" module?